tests/kj-rs/cxx-types.cc

```diff
@@ -0,0 +1,19 @@
+#include "cxx-types.h"
```

Comment:
the type is so simple you can safely drop the .cc file and have everthing in .h. less maintenance this way.